src/ProgressBar/ProgressBar.less

```diff
@@ -4,6 +4,18 @@
 	background-color: @moon-progress-bg-color;
 	direction: ltr;
 
+	.moon-neutral & {
+		background-color: @moon-neutral-progress-bar-bg-color;
```

Comment:
I think this should be @moon-neutral-progress-bg-color; to match with @moon-progress-bg-color;